### PR TITLE
[yaireo__tagify] adopt changes of core library up to version 4.18.3

### PR DESCRIPTION
**Pull request by Brakebein**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `pnpm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: [yairEO/tagify v4.16.4...v4.18.3](https://github.com/yairEO/tagify/compare/v4.16.4...v4.18.3) - reviewing all changes from 4.17.0 to latest (4.18.3)
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the `package.json`.

### CODE REVIEW

**Comment by DangerBotOSS** [comment]
## Formatting

The following files are not formatted:
1. types/yaireo__tagify/yaireo__tagify-tests.ts

Consider running `pnpm dprint fmt` on these files to make review easier.
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against e999268b4873dcd74c643a02375a2b38241fc5e1
</p>

File: types/yaireo__tagify/index.d.ts [hunk 1]
```diff
@@ -92,12 +101,25 @@ declare namespace Tagify {
...
          */
         position: DropDownPosition;
 
+        /**
+         * Dictates the dropdown's horizontal starting position. By default,
+         * it would be aligned with the left side of the Tagify component.
+         * @default false
+         */
+        RTL: boolean;
+
```
Comment (blutorange) [inline types/yaireo__tagify/index.d.ts:122]: I took another look at the tagify docs and found one more property. Could you also add the `dropdown.sortby`? If I read the docs correctly, it should be something like

```ts
/**
 * If set to `startsWith`, the suggestions list will be sorted with matched items which starts with
 * the query shown first, and exact matches shown before all.
 * 
 * If this setting is defined as a function, it receives the filtered items and the query and
 * it must return the sorted items.
 * 
 * (default sorting order is same as the whitelist's)
 */
sortby: "startsWith" | ((items: T[], query: string) => T[]);
```

**Review by blutorange — APPROVED** [review]
Looks great to me, thanks very much for keeping it up to date! I checked against the docs and the code, the types seem to be right. As mentioned in the other comment, it would be great if you could also add `sortby` property to the dropdown settings.

File: types/yaireo__tagify/index.d.ts (hunk 1 above)
Comment (Brakebein) [reply types/yaireo__tagify/index.d.ts:133]: How did we miss that one? It has been already introduced 2 years ago. Good catch! I just added this one as well.

**Review by blutorange — APPROVED** [review]
Looks all good to me!

**Comment by Brakebein** [comment]
Ready to merge

**Pull request merged**